tests/core/arch-x86/stalker-x86.c

```diff
@@ -81,6 +91,11 @@ TESTLIST_BEGIN (stalker)
 #endif
 TESTLIST_END ()
 
+#ifdef HAVE_LINUX
+static GHashTable * prefetch_compiled = NULL;
+static GHashTable * prefetch_executed = NULL;
+#endif
```

Comment:
Variables should go below function prototypes. So these should be moved to the bottom of the `HAVE_LINUX` block introduced below, with a blank line separating this and the function prototypes.

Reply:
Ok.